end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit. This is because [zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA if multiple changes are running in the gate. Since the gate queue tests multiple changes together in their theoretical merge order, the change in SHA for one commit will confuse zuul when it begins to test commits that rely on the commit that has been changed.
```

Comment:
Can we simplify this? I understand what it is meaning, and my eyes are glazing over...

@jesusaurus, and @j2sol: what are your thoughts?